sites/website/src/docs/introduction.md

```diff
@@ -17,13 +17,13 @@ FAST is a collection of technologies built on Web Components and modern Web Stan
+One of FAST's driving principals is "strive to adopt open, web standards-based approaches as much as possible." To that end, FAST is built directly on the W3C Web Component standards mentioned above, and does not create its own component model. This allows components built with FAST to function the same as built-in, native HTML elements. You do not need a framework to use FAST components, but you can use them in combination with any framework or library of your choice.
 
 ### How can FAST help me?
 
 To understand how FAST can help you, let's take a look at the FAST tech stack from top to bottom.
 
-At the top of the stack FAST provides two sets of Web Components: `@fluentui/web-components` and `@microsoft/fast-components`. Both of these libraries include a common set of components found in many websites and apps. Example components include button, menu, tree-view, tab, card, modal, and more. So, what is the difference between these two libraries? The answer is that each library implements a different *design system*. `@fluentui/web-components` implements Microsoft's Fluent Design System. They look and feel like the components found in products like Windows, Office, Teams, and Edge. If you like the Fluent Design System or are building experiences designed to fit naturally into the Microsoft ecosystem, this is the component set you'll want to use. Alternatively, `@microsoft/fast-components` provides an industry-focused design system with more customization options. We call this system "FAST Frame". If you're looking to integrate FAST components into an existing site or app, or if you need more control over the theme of the components, this is the option you'll want to start with.
+At the top of the stack, FAST provides two sets of Web Components: `@fluentui/web-components` and `@microsoft/fast-components`. Both of these libraries include a common set of components found in many websites and apps. Example components include button, menu, tree-view, tab, card, modal, and more. So, what is the difference between these two libraries? The answer is that each library implements a different *design system*. `@fluentui/web-components` implements Microsoft's Fluent Design System. They look and feel like the components found in products like Windows, Office, Teams, and Edge. If you like the Fluent Design System or are building experiences designed to fit naturally into the Microsoft ecosystem, this is the component set you'll want to use. Alternatively, `@microsoft/fast-components` provides an industry-focused design system with more customization options. We call this system "FAST Frame". If you're looking to integrate FAST components into an existing site or app, or if you need more control over the theme of the components, this is the option you'll want to start with.
```

Comment:
Unrelated to this, but @eisenbergeffect we may want to workshop this a bit. We technically maintain both of these currently but I think we should potentially focus this on _just_ FAST and not Fluent, as it's technically not the same offering. Again, unrelated here and a larger change, but wanted to bring attention to it.

Reply:
@chrisdholt I talked with @huynhicode last week about a future task to completely rework our introduction. So, we'll have a separate PR for that at some point that we can jam on. I'd like to have some visuals for that and @huynhicode had some idea to introduce some more color.

I also think we should rework the home page. Because if we remove Fluent from here, I think we should consider having a section of our home page that calls out how FAST can be used to build design systems. In that section we can link out to both of our systems and even have a few links out to other systems like Furious and VS Code. Now that we've got some official Fluent docs out, I don't think we need Fluent info spread throughout our FAST docs, but I do think there should be some sort of connection from FAST to Fluent. The home page and a section on design systems built on FAST might be a better place to do that.